api/src/middleware/auth.ts

```diff
@@ -0,0 +1,17 @@
+import jwt from "jsonwebtoken";
+import env from "../../environment";
+
+export default function auth(req, res, next) {
+    const authHeader = req.header("authorization");
+    if (!authHeader) {
+        return res.status(401).json({ msg: "No token, authorization denied." });
+    }
+    const token = authHeader.split(" ")[1];
```

Comment:
```suggestion
    const token = req.get("Authorization");
    
    if (ValidationUtils.isEmpty(token)) {
        return res.status(401).json({ msg: "No token, authorization denied." });
    }
```

Acho que assim fica mais conciso

Reply:
Beleza!